doc/api/diagnostics_channel.md

```diff
@@ -1117,6 +1117,64 @@ Emitted when server receives a request.
 
 Emitted when server sends a response.
 
+#### Modules
+
+`module.require.start`
+
+* `event` {Object} containing the following properties
+  * `id` - Argument passed to `require()`. Module name.
+  * `name` - Event name.
+  * `parentFilename` - Name of the module that attempted to require(id).
+
+Emitted when `require()` is executed
+
+`module.require.end`
+
+* `event` {Object} containing the following properties
+  * `id` - Argument passed to `require()`. Module name.
+  * `name` - Event name.
+  * `parentFilename` - Name of the module that attempted to require(id).
+
+Emitted when a `require()` call returns
```

Comment:
```suggestion
Emitted when a `require()` call returns.
```
Is it always called on end of `require` or only in case of success?

Reply:
Only in case of success.